doc/_extensions/zephyr/domain.py

```diff
@@ -80,6 +80,9 @@ class ConvertCodeSampleNode(SphinxTransform):
     default_priority = 100
 
     def apply(self):
+        if not self.config.zephyr_domain_apply_transforms:
```

Comment:
I'm sure you did it this way for a reason, but wouldn't it be simpler/clearer to simply not install the transforms in the `setup()`? Also see my general comment/question to understand better what this is trying to accomplish

Reply:
I'm not sure how to do that from setup() (Sphinx settings are defined there)

Reply:
I mean just don't add them to app at lines L317-L321?